test/units/param_helpers_test.rb

```diff
@@ -11,10 +11,14 @@ def test_param_helpers_emits_dep_warning
 
     assert_output(nil, host_warn) {
         client = Imgix::Client.new(host: 'test.imgix.net')
+        prefix_warn = "Warning: `Client::prefix' will take zero arguments " \
```

Comment:
I just realized that we are also erroneously passing in an argument to `prefix` within `Imgix::Client` [[1](https://github.com/imgix/imgix-rb/blob/cd9ddf10a91a8d80bff532b5b730bc2edeb47b73/lib/imgix/client.rb#L39)][[2](https://github.com/imgix/imgix-rb/blob/cd9ddf10a91a8d80bff532b5b730bc2edeb47b73/lib/imgix/client.rb#L48)]. That makes me nervous because we will definitely be emitting warnings to users when they call `Imgix::path` without giving them a way to resolve it.

Perhaps it would be better if we refactor our own code to not pass in an argument to `prefix` and then explicitly call it in our tests?

Reply:
Yeah, I don't like this either. I have a different patch I should've pushed. One sec.